test/e2e/zoneegress/externalservices/zoneegress_externalservices_hybrid.go

```diff
@@ -128,124 +116,57 @@ networking:
 			KumactlApplyFromString(
 				fmt.Sprintf(externalService2, nonDefaultMesh, net.JoinHostPort(zone4.GetApp("es-test-server").GetIP(), "8080"))),
 		).To(Succeed())
 	})
 
-	It("k8s should access external service through zoneegress", func() {
-		filter := fmt.Sprintf(
-			"cluster.%s_%s.upstream_rq_total",
-			nonDefaultMesh,
-			"external-service-1",
-		)
-
-		clientPodName, err := PodNameOfApp(zone1, "demo-client", TestNamespace)
-		Expect(err).ToNot(HaveOccurred())
-
-		Eventually(func(g Gomega) {
-			stat, err := zone1.GetZoneEgressEnvoyTunnel().GetStats(filter)
-			g.Expect(err).ToNot(HaveOccurred())
-			g.Expect(stat).To(stats.BeEqualZero())
-		}, "30s", "1s").Should(Succeed())
-
-		_, stderr, err := zone1.ExecWithRetries(TestNamespace, clientPodName, "demo-client",
-			"curl", "--verbose", "--max-time", "3", "--fail", "external-service-1.mesh")
-		Expect(err).ToNot(HaveOccurred())
-		Expect(stderr).To(ContainSubstring("HTTP/1.1 200 OK"))
-
-		Eventually(func(g Gomega) {
-			stat, err := zone1.GetZoneEgressEnvoyTunnel().GetStats(filter)
-			g.Expect(err).ToNot(HaveOccurred())
-			g.Expect(stat).To(stats.BeGreaterThanZero())
-		}, "30s", "1s").Should(Succeed())
+	E2EAfterAll(func() {
+		Expect(zone1.DeleteNamespace(TestNamespace)).To(Succeed())
+		Expect(zone1.DeleteKuma()).To(Succeed())
+		Expect(zone1.DismissCluster()).To(Succeed())
+		Expect(zone4.DismissCluster()).To(Succeed())
+		Expect(global.DismissCluster()).To(Succeed())
 	})
 
-	It("universal should access external service through zoneegress", func() {
-		filter := fmt.Sprintf(
-			"cluster.%s_%s.upstream_rq_total",
-			nonDefaultMesh,
-			"external-service-2",
-		)
-
-		Eventually(func(g Gomega) {
-			stat, err := zone4.GetZoneEgressEnvoyTunnel().GetStats(filter)
-			g.Expect(err).ToNot(HaveOccurred())
-			g.Expect(stat).To(stats.BeEqualZero())
-		}, "30s", "1s").Should(Succeed())
-
-		stdout, _, err := zone4.ExecWithRetries("", "", "zone4-demo-client",
-			"curl", "--verbose", "--max-time", "3", "--fail", "external-service-2.mesh")
-		Expect(err).ToNot(HaveOccurred())
-		Expect(stdout).To(ContainSubstring("HTTP/1.1 200 OK"))
-
-		Eventually(func(g Gomega) {
-			stat, err := zone4.GetZoneEgressEnvoyTunnel().GetStats(filter)
-			g.Expect(err).ToNot(HaveOccurred())
-			g.Expect(stat).To(stats.BeGreaterThanZero())
-		}, "30s", "1s").Should(Succeed())
-	})
-
-	It("k8s should not reach external service when zone egress is down", func() {
-		// given k8s cluster
-		k8sCluster := zone1.(*K8sCluster)
-
-		clientPodName, err := PodNameOfApp(zone1, "demo-client", TestNamespace)
-		Expect(err).ToNot(HaveOccurred())
+	Context("passthrough false with zoneegress false", func() {
+		BeforeAll(func() {
+			Expect(YamlUniversal(fmt.Sprintf(meshMTLSOn, nonDefaultMesh, "false", "false"))(global)).To(Succeed())
+		})
 
-		serviceUnreachable := func() error {
-			_, _, err := k8sCluster.Exec(TestNamespace, clientPodName, "demo-client",
+		It("k8s should reach external service", func() {
```

Comment:
I believe there is a convention to name it `It("should ...` so in these cases:
* `It(should reach external service from k8s zone)`
* `It(should reach external service from universal zone)`